backend/spherre/tests/test_models/test_transaction_model.py

```
@@ -0,0 +1,27 @@
+        self.app = create_app()
+        self.client = self.app.test_client()
+        self.ctx = self.app.app_context()
+        self.ctx.push()
+        db.create_all()
+
+    def tearDown(self):
+        db.session.remove()
+        db.drop_all()
+        self.ctx.pop()
+
+    def test_create_transaction_model():
```

Pylint (3.3.7)

[error] Method 'test_create_transaction_model' has no argument

(E0211)